tests/Makefile

```diff
@@ -36,7 +47,7 @@ found_apt := $(shell command -v apt 2>/dev/null)
 found_bats := $(shell command -v bats 2>/dev/null)
 found_npm := $(shell command -v npm 2>/dev/null)
 
-found_podman := $(shell command -v podman 2>/dev/null)
+found_podman := $(shell docker --version 2>/dev/null | grep podman &>/dev/null)
```

Comment:
Does the podman binary always shadow `docker`?

Reply:
If I recall correctly. I'll double check.

Reply:
Reworked the checks entirely now so that should improve the remark below as well.